.github/actions/maven-build-and-tag/action.yml

```
@@ -89,7 +89,7 @@ runs:
           ${{ runner.os }}-maven-
 
     - name: Set up JDK
-      uses: actions/setup-java@0ab4596768b603586c0de567f2430c30f5b0d2b0 # v3.13
+      uses: actions/setup-java@387ac29b308b003ca37ba93a6cab5eb57c8f5f93 # v3.13
```

```suggestion
      uses: actions/setup-java@387ac29b308b003ca37ba93a6cab5eb57c8f5f93 # v4.0.0
```

@Giovanni007 @atchertchian can be a glitch in the script which has migrated tag to sha?

---

I think the dependabot bug is linked to the fact that version should be `v3.13.0`, not `v3.13`, and when I run the script myself I do get v3.13,0 so not sure why, wdyt @Giovanni007 ?

Side note: my diff on current master branch when running the script
```
iff --git a/.github/actions/maven-build-and-tag/action.yml b/.github/actions/maven-build-and-tag/action.yml
index 48fdc47..40826db 100644
--- a/.github/actions/maven-build-and-tag/action.yml
+++ b/.github/actions/maven-build-and-tag/action.yml
@@ -89,7 +89,7 @@ runs:
           ${{ runner.os }}-maven-
 
     - name: Set up JDK
-      uses: actions/setup-java@0ab4596768b603586c0de567f2430c30f5b0d2b0 # v3.13
+      uses: actions/setup-java@0ab4596768b603586c0de567f2430c30f5b0d2b0 # v3.13.0
       with:
         java-version: ${{ inputs.java-version }}
         distribution: ${{ inputs.java-distribution }}
diff --git a/.github/actions/send-teams-notification/action.yml b/.github/actions/send-teams-notification/action.yml
index 751b64f..a78791a 100644
--- a/.github/actions/send-teams-notification/action.yml
+++ b/.github/actions/send-teams-notification/action.yml
@@ -63,7 +63,7 @@ runs:
       uses: martialonline/workflow-status@fe13c6a4716673e224038aa1b02387352fb35e13
       id: workflow_status_check
     - name: Send teams notification
-      uses: skitionek/notify-microsoft-teams@9c67757f64d610fb6748d8ff3c11f284355ed7ec # v1.0
+      uses: skitionek/notify-microsoft-teams@9c67757f64d610fb6748d8ff3c11f284355ed7ec # v1.0.8
       with:
         webhook_url: ${{ inputs.webhook-url }}
         job: ${{ toJSON(steps.workflow_status_check.outputs) }}
diff --git a/.github/workflows/terraform-eks.yml b/.github/workflows/terraform-eks.yml
index 125cc6d..43eed9f 100644
--- a/.github/workflows/terraform-eks.yml
+++ b/.github/workflows/terraform-eks.yml
@@ -75,7 +75,7 @@ jobs:
         ./aws/install
     steps:
       - name: Checkout
-        uses: actions/checkout@v4
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
       - name: Load environment variables from yml

```

---

I have opened at PR for it already @gionn : https://github.com/Alfresco/alfresco-build-tools/pull/423
We should bump the check action because it did not notice the missing sha in terraform-eks.yml

---

and I'm not sure why @Giovanni007 did not have the same version of the script as afaik it did not change after he started